staging/src/k8s.io/api/core/v1/well_known_taints.go

```diff
@@ -49,4 +49,10 @@ const (
 	// TaintNodeOutOfService can be added when node is out of service in case of
 	// a non-graceful shutdown
 	TaintNodeOutOfService = "node.kubernetes.io/out-of-service"
+
+	// TaintExternalCloudProvider sets this taint on a node to mark it as unusable,
+	// when kubelet is started with the "external" cloud provider, until node-controller
+	// from the cloud-controller-manager or cloud-node-manager intitializes this node,
+	// and then node-lifecycle-controller removes the taint
+	TaintExternalCloudProvider = "node.cloudprovider.kubernetes.io/uninitialized"
```

Comment:
suggest a name like `TaintNodeCloudProviderUninitialized`

Reply:
it actually looks like this is already present in this repo in https://github.com/kubernetes/kubernetes/blob/master/staging/src/k8s.io/cloud-provider/api/well_known_taints.go#L24

Reply:
Ah. OK, will reuse the existing one.